pkg/application/controller/app/action/install.go

```diff
@@ -81,16 +81,23 @@ func Install(ctx context.Context,
 		return nil, err
 	}
 
+	var clientTimeout = defaultTimeout
```

Comment:
is this nessary?

Reply:
why not? Usually we don't set timeout in installPara or upgradePara, and we use the defaultTimeout.

Reply:
ok